beanstalk_worker/views.py

```diff
@@ -22,3 +25,54 @@ def run_all(request):
     count = task_service.run_all()
 
     return HttpResponse(f"Ran {count} tasks")
+
+
+@csrf_exempt
+def task_launcher(request, task_name, user_name):
+    try:
+        the_user = User.objects.get(username=user_name)
+    except User.DoesNotExist:
+        return JsonResponse({"status": "fail", "error": f"User not found for {user_name}"})
+    except Exception as e:
+        return JsonResponse(
```

Comment:
add a logging.exception here so we get an error in sentry otherwhise we will miss all the error as they are catched.

Add a status code as well, like 400?